src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -1966,9 +1966,13 @@ private String getResourceFile(final String uri, final String contentType) throw
         return result;
     }
 
-    private void storePluginPropertiesAsParams(final Map<String, String> pluginProperties, final Multimap<String, String> params) {
+    protected void storePluginPropertiesAsParams(final Map<String, String> pluginProperties, final Multimap<String, String> params) {
         for (final String key : pluginProperties.keySet()) {
             params.put(JaxrsResource.QUERY_PLUGIN_PROPERTY, String.format("%s=%s", UTF8UrlEncoder.encode(key), UTF8UrlEncoder.encode(pluginProperties.get(key))));
         }
     }
+
+    protected KillBillHttpClient getKillBillHttpClient(){
```

Comment:
@pierre this was supposed to be another PR, we tried to make KillBillClient extendable to add more methods, this why I added `protected` to make the client and the utility methods accessible to child classes. should I move it to another branch?